src/commons/utils/InfiniteLoopReporter.ts

```diff
@@ -78,7 +70,7 @@ function reportPotentialInfiniteLoop(
 function reportNonErrorProgram(sessionId: number, code: string[]) {
   Sentry.withScope(function (scope) {
     scope.clearBreadcrumbs();
-    scope.setLevel(Sentry.Severity.Info);
+    scope.setLevel('info');
```

Comment:
Should we make this a constant in our repo instead?

Reply:
I don't think this will be necessary. Seems like this is the 'canonical' way of setting the severity level in sentry (see [their docs](https://docs.sentry.io/platforms/javascript/usage/set-level/)). 

Also, the current (depreciated) `Sentry.Severity.Info` is already a constant enum] ([source](https://github.com/getsentry/sentry-javascript/blob/master/packages/types/src/severity.ts)).